lib/index.ts

```diff
@@ -7,14 +7,14 @@ import {
   ProperCycle,
   ProperCycles,
   PSALTER_WEEKS,
-  PsalterWeeksCycle,
-  PsalterWeeksCycles,
+  PsalterWeekCycle,
+  PsalterWeekCycles,
+  SundayCycle,
+  SundayCycles,
   SUNDAYS_CYCLE,
```

Comment:
Is the name of `SUNDAYS_CYCLE` and `WEEKDAYS_CYCLE` (L17) intentional? Didn’t you want to rename them into singular?

Reply:
Yes, they should be renamed to `SUNDAY_CYCLES` and `WEEKDAY_CYCLES` as well 👍  
They are arrays of all possible cycles in their categories.